remoting/subsystem/src/main/java/org/jboss/as/remoting/HttpConnectorAdd.java

```diff
@@ -51,6 +52,14 @@ private HttpConnectorAdd() {
     }
 
     @Override
+    protected void populateModel(OperationContext context, ModelNode operation, Resource resource) throws OperationFailedException {
+        super.populateModel(context, operation, resource);
+        final PathAddress address = context.getCurrentAddress();
+        final String connectorName = address.getLastElement().getValue();
+        validateDuplicatedResources(context, address, connectorName, resource);
```

Comment:
This needs to be wrapped in an added step so it executes after other ops in the same composite, since those other ops might correct the condition the validator will reject.